modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,320 @@
++
+|===
+| Regular Node | Importer Node
+
+| link:{attachmentsdir}/kubernetes/sgw-deployment.yaml[sgw-deployment.yaml]
+| link:{attachmentsdir}/kubernetes/sgw-deployment-import.yaml[sgw-deployment-import.yaml]
+|===
+The following example shows the controller file for a regular node.
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sync-gateway # <1>
+spec:
+  replicas: 2 # <2>
+  template:
+    metadata:
+      labels:
+        app: sync-gateway
+    spec:
+      containers:
+        - name: sync-gateway
+          image: couchbase/sync-gateway:2.1.2-enterprise # <3>
+          args: ["/sync-gateway-config/sgw-config.json"] # <4>
+          volumeMounts: # <5>
+            - name: sgw-config-volume
+              mountPath: /sync-gateway-config
+              readOnly: true
+          env:
+            - name: GOMAXPROCS # <6>
+              value: "2"
+          resources:
+            requests:
+              cpu: "2"
+            limits:
+              cpu: "2" # <7>
+      volumes: # <8>
+        - name: sgw-config-volume
+          secret:
+            secretName: sgw-config
+----
+<1> `metadata.name`: The name of the deployment is "sync-gateway".
+<2> `spec.replicas`: 2 Sync Gateway replicas that are deployed at most.
+* For import node deployment, this *must* be 1 because there can be at most 1 Sync Gateway replicator in a cluster.
+If high availability is important, you probably want at least 2 Sync Gateway import nodes.
+* For replicator node deployment, this *must* be 1 because there can be at most 1 Sync Gateway import node in a cluster.
+<3> `containers[].image`: Points to the docker image for Sync Gateway.
+<4> `containers[].args`: Points to the Sync Gateway configuration file named "sgw-config-working.json" which is mounted at the path specified via the `volumeMounts` config.
+<5> `volumeMounts`: Specifies where to mount the volume into the container.
+<6> `GOMAXPROCS`: This GO runtime environment variable is used to limit the number of system threads that are allocated to Sync Gateway.
+<7> `containers[].resources.limits.cpu`: This is used to specify the CPU limit for the Sync Gateway pod.
+If you do not specify one, the Sync Gateway could spawn as many processes as CPU cores and potentially use up all CPU resources.
+You can learn more about CPU resource assignment https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#if-you-do-not-specify-a-cpu-limit[here].
+We recommend a value of 2 but you should use what is suited for your environment.
+<8> `volumes`: Specifies what to mount.
+In our case, the "secret" with name "sgw-config" corresponding to the Sync Gateway configuration that was created in the previous step is mounted.
+Learn more about Kubernetes volumes https://kubernetes.io/docs/concepts/storage/volumes/[here].
+. Deploy the Sync Gateway cluster from the specified deployment controller file.
++
+[source,console]
+----
```

Comment:
@jamiltz  Same as before. Please include separate tabs for import and regular with exact command that users can copy and paste

Reply:
https://github.com/couchbase/docs-sync-gateway/commit/a230695a63497984aad1f70dcec4f393d55f0804